client/deposits/index.js

```diff
@@ -19,27 +19,29 @@ import ListBanner from '../empty-state-table/deposits-banner.svg';
 import { Experiment } from '@woocommerce/explat';
 
 const DepositsPage = () => {
+	const defaultExperience = (
+		<>
+			<TestModeNotice topic={ topics.deposits } />
+			<DepositsList />
+		</>
+	);
+
+	const treatmentExperience = wcpaySettings.accountStatus.status ? (
```

Comment:
This condition is OK for "WCPay not connected". 

However, the original issue https://github.com/Automattic/woocommerce-payments/issues/1816 mentions: 

> As a merchant who has installed WCPay but not completed the KYC. 

If so, two statuses `restricted` and `restricted_soon` will fall into this condition too? cc @moon0326 and @mattsherman

Here is the reference to these two statuses in the Overview page https://github.com/Automattic/woocommerce-payments/blob/2f24127eb990fd79ed2bcf6d56ba3d534f1e7774/client/account-status-settings/index.js#L46-L76

Reply:
I'm pretty sure we need to show these records even if the account is currently in a restricted state (or will be soon), but that's a great question in terms of the onboarding changes imminent on the server, since there will be accounts linked that have not yet completed initial KYC. I don't think they will be considered connected from the plugin's perspective until the initial KYC details are submitted, but we should check on that.

Reply:
Confirmed that with the new onboarding changes, there is currently no account status being returned until WCPay is connected _and onboarded_, so AFAIK this check will do.

Reply:
If so, I think we should be very clear about two terms "not yet completed KYC" and "WCPay not connected" as they're not the same. Otherwise, it will be very confusing for all people involved. 

If I understand correctly: 

- WCPay not connected = No KYC provided.
- Not yet completed KYC = (1) WCPay connected AND (2) some KYC docs provided but not enough = two statuses `restricted` or `restricted_soon` 

If this is correct, I will raise a small discussion in the upstream - this PT paJDYF-1rg-p2

Reply:
`restricted` means there are requirements that are past due, and `restricted_soon` means there are requirements that will be due soon – for our current purposes, these would happen after initial KYC. (The status is defined in the `get_account_status` server function, and it's intended to match the Stripe dashboard's account status – e.g. [here](https://stripe.com/docs/connect/dashboard#restricted) – but isn't _directly_ derived from it.)

I think if we were to modify the server to return the account info prior to completion of initial KYC, we would assign a new status for that state. At that point, we can modify this logic to exclude that state _if_ we choose to keep this preview mode until initial KYC is complete, but we don't want to hide data just because there are currently (or soon will be) unfulfilled requirements on the account – does that sound right?

Reply:
@dechov - your reply makes sense and I totally agree. However, that also means we may need to update this P2 paJDYF-1rg-p2 and ask for using the correct wording "WCPay not connected". 

I am going to add a comment in this P2.

Reply:
Comment paJDYF-1rg-p2#comment-7381